Wire-iOS/Sources/Helpers/URL+Wire.swift

```diff
@@ -45,6 +45,14 @@ extension URL {
 
 extension URL {
 
+    static var wr_itunes: URL {
+        return URL(string: "https://itunes.apple.com/us/app/wire/id930944768?mt=8")!
```

Comment:
While we are at it. As Apple recommends in the Link Maker, maybe it would be more resilient to use a global link here, because it automatically redirects the user to the correct storefront (eg, if they're not on the US store, they will be redirected to the DE store). cc @marcoconti83 

```suggestion
        return URL(string: "https://geo.itunes.apple.com/us/app/wire/id930944768?mt=8")!
```

Reply:
That sounds like a good idea I think!